lidar_yolo_model:

  config_file:
    name: yolov3-tiny.cfg
  weight_file:
    name: yolov3-tiny_50000_lidar.weights
  threshold:
    value: 0.3
  detection_classes:
    names:
      - Car
      - Van
      - Truck
      - Pedestrian
      - Person_sitting
      - Cyclist
      - Tram
      - Misc
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
      - DontCare
